zomes/test/src/lib.rs

```diff
@@ -16,20 +16,73 @@ fn target() -> ExternResult<EntryHash> {
     path("b")
 }
 
+#[hdk_extern]
+fn init(_: ()) -> ExternResult<InitCallbackResult> {
+    set_read_only_cap_tokens()?;
+
+    Ok(InitCallbackResult::Pass)
+}
+
+pub fn set_read_only_cap_tokens() -> ExternResult<()> {
+    let mut functions: GrantedFunctions = BTreeSet::new();
+    functions.insert((zome_info()?.name, "returns_obj".into()));
+    functions.insert((zome_info()?.name, "pass_obj".into()));
+    functions.insert((zome_info()?.name, "return_failure".into()));
+    functions.insert((zome_info()?.name, "create_link".into()));
+    functions.insert((zome_info()?.name, "delete_link".into()));
+    functions.insert((zome_info()?.name, "get_link".into()));
+    functions.insert((zome_info()?.name, "delete_all_link".into()));
+    functions.insert((zome_info()?.name, "signal_loopback".into()));
+    create_cap_grant(CapGrantEntry {
+        tag: "".into(),
+        access: ().into(),
+        functions,
+    })?;
+    Ok(())
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes, Clone)]
+pub struct Props {
+    pub skip_proof: bool,
+}
+
+pub fn skip_proof_sb(encoded_props: &SerializedBytes) -> bool {
+    let maybe_props = Props::try_from(encoded_props.to_owned());
+    if let Ok(props) = maybe_props {
+        debug!("Using props : {:?}", props);
```

Comment:
I think we want to remove this log or make it more specific

If we keep it, something like

```suggestion
        debug!("Skip proof check. Props: {:?}", props);
```